src/main/resources/lib/autologin.ts

```diff
@@ -0,0 +1,20 @@
+/* global log, __ */
+import type {Request} from '@item-enonic-types/global/controller';
+
+import {extractJwtToken} from './jwt';
+
+export const autoLogin = function (req: Request) {
+    try {
+        const jwtToken = extractJwtToken(req);
+        if (!jwtToken) {
+            return false;
+        }
+        return __.newBean<{
+            verifyAndLogin(jwtToken: string): boolean;
+        }>('com.enonic.app.standartidprovider.handler.JwtHandler').verifyAndLogin(jwtToken);
```

Comment:
Can we fix a typo in the java package name (`standartidprovider` vs `standardidprovider`)?

Reply:
Hehe, good one.